.github/workflows/ci.yml

```diff
@@ -50,3 +50,7 @@ jobs:
           APPLE_DEVELOPMENT_SIGNING_CERTIFICATE: ${{ secrets.APPLE_DEVELOPMENT_SIGNING_CERTIFICATE }}
           APPLE_DEVELOPMENT_SIGNING_P12_PASSWORD: ${{ secrets.APPLE_DEVELOPMENT_SIGNING_P12_PASSWORD }}
           EXTRA_XCODEBUILD: ${{ env.EXTRA_XCODEBUILD }}
+      
+      - name: Run Unit Tests
+        if: matrix.platform == 'macOS'
```

Comment:
We do not need to run them twice (iOS + macOS),
they are going to be the same tests as of now.
The macOS will run significantly faster, as it won't require to set up the simulator.

Reply:
I don't think that's a good idea. The point of CI is to trap unexpected regressions. With this we might miss an iOS only regression. And we'll never think of changing this when growing tests.
I don't think slightly slower CI runs are an issue

Reply:
Ok, I've updated it to run it on both platforms.